From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Franz-Josef Haider <franz.haider@jolla.com>
Date: Mon, 12 Oct 2020 17:11:00 +0000
Subject: [PATCH] (hybris) create /dev/socket during first stage

It will attempt to create it again during 2nd stage, do not fail then.

Signed-off-by: Simonas Leleiva <simonas.leleiva@jolla.com>
Change-Id: I4701723d55032e35228a84bff81b20cc93424c32
---
 init/first_stage_init.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/first_stage_init.cpp b/init/first_stage_init.cpp
index f2d6c8ace224ecb690ddfd636d3cbca232f2b2b0..5e485238dcc0f59bc4befd3c2fbe05f2deae43a7 100644
--- a/init/first_stage_init.cpp
+++ b/init/first_stage_init.cpp
@@ -237,7 +237,7 @@ int FirstStageMain(int argc, char** argv) {
     // on / and then we'll let the rc file figure out the rest.
     //CHECKCALL(mount("tmpfs", "/dev", "tmpfs", MS_NOSUID, "mode=0755"));
     //CHECKCALL(mkdir("/dev/pts", 0755));
-    CHECKCALL(mkdir("/dev/socket", 0755));
+    /*CHECKCALL*/(mkdir("/dev/socket", 0755));
     CHECKCALL(mkdir("/dev/dm-user", 0755));
     //CHECKCALL(mount("devpts", "/dev/pts", "devpts", 0, NULL));
 #define MAKE_STR(x) __STRING(x)
